CSharp/core-DirectLineWebSockets/DirectLineClient/packages.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Bot.Connector.DirectLine" version="3.0.0" targetFramework="net461" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.4" targetFramework="net461" />
```

Comment:
We've hit binary incompatibilities between minor versions of Microsoft.Rest.ClientRuntime. 2.3.2 is probably a better example to use.